### ISSUE TITLE
Bans

### ISSUE THREAD

**Issue by MunchnetNL** [Phaetec/pogo-cruncher#78]
There are warnings due to permanent banstaff. 

I crunched pokemon twice after the post on reddit. One while not aware of 284 pokemon latest was 2 hours ago of 151 pokemon while aware of the warning post. 
So far no ban for me.

**Comment by MunchnetNL** [comment]
Still no ban

**Comment by Wegi** [comment]
Me neither. Haven't read of anyone using those kind of tools getting banned yet. I think it should be safe now. I'll wait till evening until i remove / change the warning.

**Comment by MunchnetNL** [comment]
Let''s turn this into an enhancement request. 

Would it be possible to have the warning show on index file we open. 
For example: have an iframe that loads a remote blank html file and should there be another warning then you could change that blank html file so it contains a warning and appears in the index file of current pogo-cruncher version installed.
The first time I crunched pokemon yesterday the warning was there on github but I only checked releases so I missed it. 
If it would have loaded in the index file of pogo-cruncher then it would have seen before login.
This potentially warns people sufficiently. 

(Yes I am noob when it comes to coding, hope the idea is clear though. Any other more efficient way is also welcome)

**Comment by Wegi** [comment]
Yeah i get the hang of it, a dynmaically loading test would be possible I guess. Will take a look later on.

**Comment by MunchnetNL** [comment]
Well I really mean a way for you @Wegi to push notifications to "currently installed" (once implemented and installed ofc) 

with this you could not only notify of ban warnings but also push notifications on new releases and or changelog.

**Comment by Geekatori** [comment]
I didn't use the cruncher (even to get IV) since last week. Any news about ban?

**Comment by MunchnetNL** [comment]
Been using it every day, crunched over 500 pokemon in the past 2 days. 
Didn't even get a softban or anything. 
So personally I would say that this is save to use at the moment.

**Comment by Wegi** [comment]
Thank you for the feedback, this is also my experience.

**Comment by n2o** [comment]
I used our Tool and didn't receive a ban. But I only use it for crunching.

**Comment by greedseed** [comment]
Have bin using the software every day. Still no ban! 

I do make sure that my phone is not running Pogo at the time that i run Cruncher.
